### DISCUSSION TITLE
Is it possible to exchange two adjacent buffers shown by tabby.nvim?

### DISCUSSION THREAD

**Discussion by nyngwang** [nanozuki/tabby.nvim#95] [Q&A] [answered]
As title. I'm a new user from bufferline.nvim.

**Comment by nanozuki** [comment, marked as answer]
Do you mean to exchange two tabs? Or exchange two windows/buffers in one tab?

**Reply by nyngwang** [reply]
The latter one, please~

To be specific: I mean the horizontal list on the right-hand-side of the preview img in your README.md.

**Reply by nanozuki** [reply]
OK, I see... It‘s the order of neovim windows in the current tab. So, If you want to change the order, you can exchange the windows' position by pressing `<C-w> h/j/k/l`. 

If you want to change the order but not change the windows' position, use `local wins = line.wins()` to write customer window labels.

**Reply by nyngwang** [reply]
Could you elaborate more on the the line `local wins = line.wins()`: Did you mean that there is an API to pass into so as to update the ordering?
